python/air/dialects/_air_ops_ext.py

```diff
@@ -106,30 +107,29 @@ class ChannelGet(ChannelGetOp):
     def __init__(
         self,
         chan_name,
-        indices,
         dst,
         dst_offsets=[],
         dst_sizes=[],
         dst_strides=[],
+        indices=[],
         async_token=None,
         async_dependencies=[],
         loc=None,
         ip=None,
     ):
-        iTy = IndexType.get()
-        indices = [
-            arith.ConstantOp(iTy, IntegerAttr.get(iTy, i)) if isinstance(i, int) else i
-            for i in indices
-        ]
+        indices_typed = list(map(pyint_to_index, indices))
+        dst_offsets_typed = list(map(pyint_to_index, dst_offsets))
+        dst_sizes_typed = list(map(pyint_to_index, dst_sizes))
+        dst_strides_typed = list(map(pyint_to_index, dst_strides))
         super().__init__(
             async_token=async_token,
             async_dependencies=async_dependencies,
             chan_name=chan_name,
-            indices=indices,
+            indices=indices_typed,
             dst=dst,
-            dst_offsets=dst_offsets,
-            dst_sizes=dst_sizes,
-            dst_strides=dst_strides,
+            dst_offsets=dst_offsets_typed,
+            dst_sizes=dst_sizes_typed,
+            dst_strides=dst_strides_typed,
             loc=loc,
             ip=ip,
         )
```

Comment:
I think we could drop the `dst_` and `src_` prefixes on the wrapper. The get and put ops define getOffset, etc. as part of the `ChannelInterface` ([here](https://github.com/Xilinx/mlir-air/blob/main/mlir/include/air/Dialect/AIR/AIROpBase.td#L184)) so the naming wouldn't be inconsistent with that. The prefixes on the operand names are probably there to avoid conflicting with the `ChannelInterface` but I'm not sure both are needed.

Reply:
Makes sense to me! I removed the `src_`/`dst_` prefixes